Source/WebInspectorUI/UserInterface/Controllers/JavaScriptLogViewController.js

```diff
@@ -157,22 +157,24 @@ WI.JavaScriptLogViewController = class JavaScriptLogViewController extends WI.Ob
         return consoleMessageView;
     }
 
-    updatePreviousMessageRepeatCount(count, timestamp)
+    updatePreviousMessageRepeatCount(target, count, timestamp)
     {
-        console.assert(this._previousMessageView);
-        if (!this._previousMessageView)
+        const previousMessageView = this._lastConseoleMessageViewForTargetMap.get(target);
```

Comment:
Style: we only use `const` for things that are actually truly constant (e.g. `const foo = 42;`) so please use `let` instead